sqlite3_vendor/CMakeLists.txt

```diff
@@ -33,7 +33,9 @@ if (NOT SQLite3_FOUND)
     )
 
   # The external project will install to the build folder, but we'll install that on make install.
-  install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/sqlite3_install/ DESTINATION ${CMAKE_INSTALL_PREFIX})
+  install(
+    DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/sqlite3_install/
+    DESTINATION .)
```

Comment:
All the rest of our vendor packages use `${CMAKE_INSTALL_PREFIX}` here:

* https://github.com/ros2/spdlog_vendor/blob/3cae65ab249d1ef3c9bfa203497007d8c7ef299f/CMakeLists.txt#L78
* https://github.com/ros2/yaml_cpp_vendor/blob/efe04baa5b3903fa238204f291f50f05d4fcdd18/CMakeLists.txt#L62
* https://github.com/ros2/rosbag2/blob/bfa63199421e53b8d229b9403ac7dd4ad4f155cb/sqlite3_vendor/CMakeLists.txt#L36
* https://github.com/ros2/rosbag2/blob/bfa63199421e53b8d229b9403ac7dd4ad4f155cb/shared_queues_vendor/CMakeLists.txt#L38
* https://github.com/ros2/tinyxml_vendor/blob/b7a50bf7af9f7ae8c517dd06552533d4e5f4a726/CMakeLists.txt#L37
* https://github.com/ros2/rviz/blob/7ca470831304a1cd29ec207f2c07e62ddb8552f1/rviz_assimp_vendor/CMakeLists.txt#L51
* https://github.com/ros2/rviz/blob/7ca470831304a1cd29ec207f2c07e62ddb8552f1/rviz_ogre_vendor/CMakeLists.txt#L61
* https://github.com/ros2/poco_vendor/blob/3f23e18f8d71e979e03395d860bc3053d3162ae6/CMakeLists.txt#L87
* https://github.com/ament/uncrustify_vendor/blob/0291bc988c45846400aca50cb973bf04a4de3d56/CMakeLists.txt#L63
* https://github.com/ros/resource_retriever/blob/d6a6818edeebcc3af73b77ebf88d3116d489089e/libcurl_vendor/CMakeLists.txt#L64

That doesn't make the rest of them automatically correct, but I'm not sure why this one is different.  Do we need to update all of those to use `.`, or can we just leave this one as it was?

Reply:
I did this based on this comment: https://github.com/ros2/console_bridge_vendor/pull/9#pullrequestreview-405300466

Reply:
It was changed under the impression that older CMake versions had an issue otherwise. I would guess that was won't the problem was coming from somewhere else. So referring that change sounds reasonable.